src/Sentry.Log4Net/LevelMapping.cs

```diff
@@ -15,4 +15,18 @@ internal static class LevelMapping
             _ => null
         };
     }
+
+    public static BreadcrumbLevel? ToBreadcrumbLevel(this LoggingEvent loggingLevel)
+    {
+        return loggingLevel.Level switch
+        {
+            var l when l == Level.Fatal || l == Level.Emergency || l == Level.All => BreadcrumbLevel.Critical,
```

Comment:
Having `Level.All` map to `BreadcrumbLevel.Critical` seems a bit odd. [According to the NLog Docs](https://logging.apache.org/log4net/release/sdk/html/F_log4net_Core_Level_All.htm):
>The All level designates the lowest level possible.

Would  `null`  be more suitable?

I see the code just above that `All` is being mapped to `SentryLevel.Fatal`... I imagine you tried to respect the same mappings (thanks). This is maybe more of a question for the other code owners then (@SeanFeldman @bitsandfoxes ).

Reply:
Indeed, I tried to make it consistent with `ToSentryLevel`. But I agree, mapping All to Fatal is rather weird.

> Would null be more suitable?

I think it should map to Debug. Typically such a low level should be below the appender's threshold, but if a user actually wants it, they should be able to use it for breadcrumbs. Does that make sense?

Reply:
Hm. If `null` is returned then the [`HubExtensions.AddBreadcrumb` extension](https://github.com/getsentry/sentry-dotnet/blob/470567c4882b2c13ce265ee1daa68256dc58deed/src/Sentry/HubExtensions.cs#L63) would be called without any `BreadcrumbLevel` specified and would use the default `BreadcrumbLevel` (which it turns out is [Info](https://github.com/getsentry/sentry-dotnet/blob/470567c4882b2c13ce265ee1daa68256dc58deed/src/Sentry/BreadcrumbLevel.cs#L18))

Arguably the `default` value is a good pick when no specific value has been specified. But I don't really have a strong - other than that `Critical` probably isn't the right level. I'm OK with your code returning either null (implicitly Info) or Debug.

Reply:
btw: I also raised https://github.com/getsentry/sentry-dotnet/issues/2521, which we can address in a separate commit.

Reply:
Mapped to BreadcrumbLevel.Debug ✅